Sources/Tuvi.Core.Entities/Tuvi.Core.Entities.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
-    <AnalysisLevel>none</AnalysisLevel>
+    <AnalysisLevel>6.0-minimum</AnalysisLevel>
```

Comment:
it should be "6.0-maximum"

Reply:
Could you please add [issue](https://github.com/Eppie-io/TuviCore/issues/new)?